saleor/settings.py

```diff
@@ -468,3 +473,15 @@ def get_host():
 DEFAULT_MENUS = {
     'top_menu_name': 'navbar',
     'bottom_menu_name': 'footer'}
+
+
+NOCAPTCHA = True
+
+ENABLE_RECAPTCHA = get_bool('ENABLE_RECAPTCHA', False)
+
+if 'RECAPTCHA_PUBLIC_KEY' in os.environ:
+    RECAPTCHA_PUBLIC_KEY = os.environ['RECAPTCHA_PUBLIC_KEY']
```

Comment:
What's the downside of having `None` `RECAPTCHA_PUBLIC_KEY`?
Shouldn't we preset some values for local development, or will it pass anyway?

Reply:
Hey! The issue right now is that the dependency is doing a `getattr(settings, key, test_api_key)`. So, setting to None, would make it use the key `None`.

https://github.com/praekelt/django-recaptcha/blob/34af16ba1ec8f27d0aeab1ab88d7a35a7b76b1ce/captcha/fields.py#L33

What I did think we could also do, is to set ourselves the [constants](https://github.com/praekelt/django-recaptcha/blob/develop/captcha/constants.py#L4-L5) through `environ.get`.